Makefile

```diff
@@ -1,18 +0,0 @@
-PYTHON ?= python3
-
-all:
-	echo 'nothing by default'
-
-prep_release:
-	# take previous one, and replace with the next one
-	utils/prep_release
```

Comment:
if github workflow fails somehow, how could we prep release locally? i.e. how at least bumpversion would be done?

I wonder if either Makefile or some documentation should be left behind on that.